### ISSUE TITLE
unused-fields doesn't support fragments spread

### ISSUE THREAD

**Issue by mrtnzlml** [relayjs/eslint-plugin-relay#84]
Hi! I am not sure whether this is a bug report or rather a feature request. I noticed that it's possible to spread a fragment even though it's never actually being used. I believe this rule should also catch this case since it leads to overfetching.

```patch
 ...LabelsList_data
 ...LabelsListSelect_data
 labels {
-	...Label_label   # not needed but not reported
 	id
 	name
 }
```

**Comment by poteto** [comment]
Closed by #86 and released in [1.8.0](https://www.npmjs.com/package/eslint-plugin-relay/v/1.8.0). Thanks for reporting!
